lib/std/special/compiler_rt/emutls.zig

```diff
@@ -0,0 +1,356 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2015-2018 LLVM Compiler Infrastructure
+// Copyright (c) 2020 Sebastien Marie <semarie@online.fr>
+// This file is part of [zig](https://ziglang.org/), which is MIT licensed.
+// The MIT license requires this copyright notice to be included in all copies
+// and substantial portions of the software.
+// __emutls_get_address specific builtin
+//
+// derived work from LLVM Compiler Infrastructure - release 8.0 (MIT)
```

Comment:
Are we copying old software now?

Reply:
yes, and in fact, it is on purpose. More recent releases couldn't be used for derived work: else the file couldn't be under MIT licence. So old software is still good :smile: